owa/app/js/components/common/header.jsx

```diff
@@ -47,7 +48,7 @@ export default class Header extends React.Component {
   componentWillMount() {
     apiCall(null, 'get', '/location').then((response) => {
```

Comment:
It is advisable to make api calls in `componentDidMount` instead of `componentWillMount`. You can read up on it [here](https://daveceddia.com/where-fetch-data-componentwillmount-vs-componentdidmount/)

Reply:
@andela-dbamidele I found this article very helpful, thanks for posting the link.

Reply:
I'm really glad it helped. 👍